yaksh/models.py

```diff
@@ -116,6 +124,25 @@ def activate(self):
     def deactivate(self):
         self.active = False
 
+    def add_teachers(self, *teachers):
```

Comment:
If feasible then try:
add_teacher only takes teachers with profile, so the views will call profile hasattr which will give teachers with profile to be added. And then call to the add_teacher.